src/app/pages/private/collection/collection.page.ts

```diff
@@ -181,4 +194,20 @@ export default class CollectionPage implements OnInit {
   getCollectedCardsForRarity(cardsForCurrentRarity: Card[]): Card[] {
     return cardsForCurrentRarity.filter((card: Card) => card.status === CardStatus.Collected);
   }
+
+  getCanMarkAllAsCollected() {
```

Comment:
it will be nice to avoid calling method in template, since it will be called on every cd cycle. Try to handle this logic in effect and save result to class property

Reply:
moved this value to a class property that is computed inside effect